src/bridger/Bridger.sol

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import "../interfaces/IBridger.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import "@oz/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@oz/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@oz/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
```

Comment:
you shoul pull the latest `main` and maybe just use all the latest OZ versions for the Bridger.

if you have any conflicts let me know but should be easy, i've rebased your branch to run the tests but didn't want to push the changes on your branch without asking (i can do it if you want)

Reply:
this should have latest main already

Reply:
it does not because the remappings are different, we have `@openzeppelin` and `@openzeppelin-5.0.1`

Reply:
yes, we can change the imports, just meant I dont need to pull main

Reply:
but if you `main` is up to date, importing with `@oz` should not be working...

Reply:
you are right, pulled before the merge, just rebased